src/app/components/work-item-list-entry/work-item-list-entry.component.less

```diff
@@ -89,20 +89,21 @@
       max-width: 95%;
       padding-right: 5px;
       margin-bottom: 0;
+      font-weight: 600;
       vertical-align: text-top;
     }
   }
   &-description {
     display: table-cell;
     width: 50%;
-    padding: 0 20px;
+    padding: 0 15px;
     vertical-align: middle;
     text-align: left;
   }
 }
 
-.width-10 {
-  width: 10%;
+.width-100px {
```

Comment:
See comment above about adding the `width: 100px;` to the existing class on the element.